<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDeleteMsg</class>
 <widget class="QDialog" name="DialogDeleteMsg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeleteMessage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please choose delete mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButtonDeleteAll">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Delete all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QRadioButton" name="radioButtonDeleteBefore">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>Delete messages </string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>(1-365) days ago</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="btnDelete">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="qssGroup" stdset="0">
      <string>ToolButton</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="btnCancel">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="qssGroup" stdset="0">
      <string>ToolButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
